cobrawap/__main__.py

```diff
@@ -323,35 +323,36 @@ def run_stage(profile=None, stage=None, extra_args=None, **kwargs):
     # lookup stage input file
     pipeline_config_path = config_path / 'configs' / 'config.yaml'
     config_dict = load_config_file(pipeline_config_path)
-    stage_idx = locate_str_in_list(config_dict['STAGES'], stage)
-    if stage_idx is None:
+    stage_idx_local = locate_str_in_list(config_dict['STAGES'], stage)
+    stage_idx_global = locate_str_in_list([v for k,v in stages.items()], stage)
+    if stage_idx_local is None:
         raise IndexError("Make sure that the selected stage is also specified "\
                          "in your top-level config in the list `STAGES`!")
     
     stage_config_path = get_config(config_dir=config_path / stage,
                                    config_name=f'config_{profile}.yaml',
                                    get_path_instead=True)
     
-    prev_stage = config_dict['STAGES'][stage_idx-1]
-    prev_stage_config_path = get_config(config_dir=config_path / prev_stage,
-                                        config_name=f'config_{profile}.yaml',
-                                        get_path_instead=True)
-    prev_config_name = Path(prev_stage_config_path).name
-    output_name = read_stage_output(stage=prev_stage, 
-                                    config_dir=config_path, 
-                                    config_name=prev_config_name)
-    stage_input = output_path / profile / prev_stage / output_name
+    if stage_idx_local>0:
```

Comment:
```suggestion
    if not is_first_stage:
```

Reply:
Checking `stage_idx_local>0` already does the job. The boolean variable `is_first_stage` is redundant.